tests/Http/PostTest.php

```diff
@@ -111,6 +119,8 @@ public function testCreatingAPhotoPost()
         $why_participated = $this->faker->paragraph;
         $text = $this->faker->sentence;
         $details = ['source-detail' => 'broadcast-123', 'other' => 'other'];
+        $action = factory(Action::class)->create([
+            'campaign_id' => $signup->campaign_id, ]);
```

Comment:
this formatting looks a little funky to me 😬

Reply:
i know it is weird! I had it as
```
$action = factory(Action::class)->create(['campaign_id' => $signup->campaign_id]);
```
without the `,` but StyleCi wanted me to add that! So weird!!

Reply:
hmm I wonder if something changed with our StyleCi settings?? This seems strange I can't imagine we'd really want it like this

Reply:
ah wait - I just changed it to one line and it seemed to like it better! Thanks, @katiecrane !